plugins/multilevel-linear-view/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@jbrowse/plugin-multilevel-linear-view",
+  "version": "1.7.11",
+  "description": "JBrowse 2 multilevel linear view",
+  "keywords": [
+    "jbrowse",
+    "jbrowse2"
+  ],
+  "license": "Apache-2.0",
+  "homepage": "https://jbrowse.org",
+  "bugs": "https://github.com/GMOD/jbrowse-components/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/GMOD/jbrowse-components.git",
+    "directory": "plugins/multilevel-linear-view"
+  },
+  "author": "JBrowse Team",
+  "distMain": "dist/index.js",
+  "srcMain": "src/index.tsx",
+  "main": "src/index.tsx",
+  "distModule": "dist/plugin-multilevel-linear-view.esm.js",
+  "module": "",
+  "files": [
+    "dist",
+    "src"
+  ],
+  "scripts": {
+    "build": "babel src --root-mode upward --out-dir dist --extensions .ts,.js,.tsx,.jsx",
```

Comment:
we might want to upgrade this to the tsc based build system used by other plugins, relatively new change in #3058 (might not be required for merge though)